sway/sway_text_node.c

```diff
@@ -60,6 +60,8 @@ static int get_text_width(struct sway_text_node *props) {
 	int width = props->width;
 	if (props->max_width >= 0) {
 		width = MIN(width, props->max_width);
+	} else {
+		width = MIN(width, 3840);
```

Comment:
8K monitors are 7680px large, so this seems a bit low?

Reply:
I mean, it's the boundary for the title text *before* the max_width has been set by the container code.

We should always be setting the max_width from what I can tell, so I don't think it really makes sense for us to render anything before the max width is determined. I think the code has been written as too generalized for our needs...

Reply:
Ah yeah. I suppose `max_width` isn't a great name here.